examples/vic-gov-au/package.json

```diff
@@ -11,7 +11,7 @@
     "cy:open": "cypress open",
     "cy:run": "cypress run -e TAGS='not @skip or @smoke'",
     "cy:run-smoke": "cypress run -e TAGS='@smoke' --spec 'test/e2e/integration/smoke/*'",
-    "test:dev": "cross-env BASIC_AUTH=0 start-server-and-test start:build http://localhost:3000 cy:open",
+    "test:dev": "start-server-and-test dev http://localhost:3000 cy:open",
```

Comment:
Any reason use dev to test, instead of build version?

Reply:
In production build you cant load dev tools for debugging - this command is just for running test locally and interactively